src/app/settings/admin/opa/component.ts

```diff
@@ -9,18 +9,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {Component} from '@angular/core';
+import {ChangeDetectionStrategy, Component} from '@angular/core';
 import {Context} from '@shared/components/tab-card/component';
-import {isEnterpriseEdition} from '@app/dynamic/common';
+import {DynamicTabComponent} from '@shared/components/tab-card/dynamic-tab/component';
+import {DynamicTab} from '@shared/model/dynamic-tab';
+import {Subject} from 'rxjs';
+import {takeUntil} from 'rxjs/operators';
 
 @Component({
   selector: 'km-admin-settings-opa',
   templateUrl: './template.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class AdminSettingsOPAComponent {
+  private readonly _unsubscribe = new Subject<void>();
   readonly Context = Context;
+  private _dynamicTabs = new Set<DynamicTabComponent>();
 
-  isEnterpriseEdition(): boolean {
-    return isEnterpriseEdition();
+  get dynamicTabs(): DynamicTabComponent[] {
+    return [...this._dynamicTabs];
+  }
+
+  onActivate(component: DynamicTab): void {
+    component.onTabReady.pipe(takeUntil(this._unsubscribe)).subscribe(tab => this._dynamicTabs.add(tab));
```

Comment:
I have forgotten to add `ngOnDestroy` and unsubscribe part here. Please do.

Reply:
👍